package.json

```diff
@@ -14,6 +14,7 @@
     "react-dom": "^16.3.1",
     "react-redux": "^5.0.7",
     "redux": "^3.7.2",
+    "redux-json-api": "^2.0.0-beta.5",
```

Comment:
It's nice to already use 2.0 beta version :)

Reply:
Yeah, actually it is pretty stable ;)